tsconfig.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
Please just copy tsconfig.json from either of my other projects.

https://github.com/gajus/graphql-lazyloader/blob/master/tsconfig.json

Having them consistent helps to maintain projects.